dspace/solr/oai/conf/schema.xml

```diff
@@ -176,26 +132,24 @@
    <dynamicField name="metadata.*" type="lengthfilter" indexed="true" stored="true" multiValued="true" />
  
    <!-- Dynamic fields (not used by default) -->
-   <dynamicField name="*_i"  type="int"    indexed="true"  stored="true"/>
+   <dynamicField name="*_i"  type="int"     indexed="true"  stored="true"/>
    <dynamicField name="*_s"  type="string"  indexed="true"  stored="true"/>
-   <dynamicField name="*_l"  type="long"   indexed="true"  stored="true"/>
+   <dynamicField name="*_l"  type="long"    indexed="true"  stored="true"/>
    <dynamicField name="*_t"  type="text"    indexed="true"  stored="true"/>
    <dynamicField name="*_b"  type="boolean" indexed="true"  stored="true"/>
-   <dynamicField name="*_f"  type="float"  indexed="true"  stored="true"/>
-   <dynamicField name="*_d"  type="double" indexed="true"  stored="true"/>
+   <dynamicField name="*_f"  type="float"   indexed="true"  stored="true"/>
+   <dynamicField name="*_d"  type="double"  indexed="true"  stored="true"/>
    <dynamicField name="*_dt" type="date"    indexed="true"  stored="true"/>
-   <dynamicField name="*_ti" type="tint"    indexed="true"  stored="true"/>
-   <dynamicField name="*_tl" type="tlong"   indexed="true"  stored="true"/>
-   <dynamicField name="*_tf" type="tfloat"  indexed="true"  stored="true"/>
-   <dynamicField name="*_td" type="tdouble" indexed="true"  stored="true"/>
-   <dynamicField name="*_tdt" type="tdate"  indexed="true"  stored="true"/>
-   <dynamicField name="*_pi"  type="pint"    indexed="true"  stored="true"/>
+   <dynamicField name="*_ti" type="int"     indexed="true"  stored="true"/>
+   <dynamicField name="*_tl" type="long"    indexed="true"  stored="true"/>
+   <dynamicField name="*_tf" type="float"   indexed="true"  stored="true"/>
+   <dynamicField name="*_td" type="double"  indexed="true"  stored="true"/>
+   <dynamicField name="*_tdt" type="date"   indexed="true"  stored="true"/>
+   <dynamicField name="*_pi"  type="pint"   indexed="true"  stored="true"/>
    <dynamicField name="ignored_*" type="ignored" multiValued="true"/>
    <dynamicField name="attr_*" type="textgen" indexed="true" stored="true" multiValued="true"/>
    <dynamicField name="random_*" type="random" />
    
  </fields>
  <uniqueKey>item.handle</uniqueKey>
- <defaultSearchField>item.handle</defaultSearchField>
```

Comment:
Is testing needed to confirm that the defaultSearchField and defaultOperator are not needed?

Reply:
Well caught!  these settings no longer exist, and had to be removed from the schemas, but I need to ensure that the corresponding 'df' and 'q.op' query parameters are set when needed.

Reply:
I've added code to replace these obsolete settings.

Reply:
Hmmm.  Looks now as though the proper fix would be to add these parameters as defaults in the search RequestHandler, and back out the Java code changes.